### DISCUSSION TITLE
Error requesting series

### DISCUSSION THREAD

**Discussion by shark2k** [sct/overseerr#1847] [Help] [answered]
Hello,

I opened this bug request, https://github.com/sct/overseerr/issues/1846, and it was closed as a "support" issue.
The issue is that some series (Whitstable Pearl in this case, but I checked and found a few more) are unable to be requested.
The reason given by @TheCatLady is that "Sonarr is currently a requirement for the TVDB matching feature".

I don't recall seeing anything in any of the changelogs or anywhere else about this being a requirement and I tested and I am able to request other series without issues.  I'm not really sure why certain series fail while others work fine as I would think they would all search the same provider and fallback to another provider if a result wasn't found.

I also feel like the issue should have had more explanation as to why certain series are failing like that when other series are able to be requested.  As far as I am aware, TheMovieDB is used for series information, so if the series is available on there, I don't understand why this failure would be happening.

I understand that this is a project that numerous volunteers are working on on their own time, but if you are going to close a bug report at least give more information like why the series mentioned is failing when other series are able to be requested successfully.

-Shark2k

**Comment by TheCatLady** [comment, marked as answer]
As I commented on the issue, a connected Sonarr server is required for the TVDB matching functionality.  (This has been the case since the introduction of this feature.)

Overseerr uses TMDb as its data source, but since Sonarr uses TVDB, series requests in Overseerr require a TVDB ID to be present in the TMDb data.  You can add the missing TVDB ID to the entry on TMDb, and then try submitting the request again later (keep in mind that TMDb caches stuff, so it may take a while for any changes to be take effect).

You are not seeing this issue with other series because they have TVDB IDs added to their TMDb entries.

**Reply by TheCatLady** [reply]
(The UI _does_ need to be refined/tweaked in the case that there is no Sonarr server configured, since we aren't able to display the manual match options.)

**Reply by shark2k** [reply]
What you wrote here, to me, would have been a better response to put in the issue that I opened when you closed it as it more clearly explains that the current implementation is potentially hampered by not having a Sonarr server configured.

Based on what you said, it sounds like there are some changes/updates that will need to be implemented to stop the reliance on Sonarr if a Sonarr server is not setup.  Because right now, the way I have my system setup, from my understanding of what you are saying, Sonarr shouldn't ideally have any impact on my system in the grand scheme of things.  The fact that a series can be found from within Overseerr, it should be able to be requested even without the TVDB ID not existing in TMBd because I am not using Sonarr so that shouldn't matter.  Only reason I can think that would be false though is if Overseerr is somehow still using the TVDB ID to actually get information but I don't think that's the case because I know The Honeymooners, for example, is pulling the info from TMDb because TMDb has 11 seasons (what Overseerr shows) but TVDB only has 2.

So one last thing.  Would it be correct to say, if the series media page doesn't show the TVDB icon under the network field (or at the bottom of that info box in general), then the request will most likely fail as that means there is no link in TMDb for the TVDB ID?  If so, I'll just let my users know that for the time being until this hopefully gets resolved in the future so that not having a Sonarr server setup won't prevent requests from being put in.

Thanks,
Shark2k

**Reply by TheCatLady** [reply]
It's not that the request will fail, but that that it is not possible to submit the request.  There is definitely room for us to handle this better, but for now, you can add any missing TVDB IDs to the TMDb entries and then try the request again later.

Please keep in mind that we are a small team with limited resources, so we aren't able to provide long in-depth responses to every inquiry.  It is easier for us to communicate with our user base on Discord, so I recommend that you join us there if you have questions that require detailed explanation.
